src/main/scala/ru/d4team/chat/api/PersonController.scala

```diff
@@ -0,0 +1,21 @@
+package ru.d4team.chat.api
+
+import ru.d4team.chat.services.PersonService
+import zio._
+import zio.json._
+import zio.http._
+import zio.http.model.Method
+
+trait PersonController {
+  def route: HttpApp[Any, Nothing]
+}
+
+final class PersonControllerImpl(personService: PersonService) extends PersonController {
```

Comment:
Consider using a case class

For more information, see https://zio.dev/reference/service-pattern/#2-service-implementation

Reply:
Fixed it